mods/tuxemon/db/item/apple.json

```diff
@@ -4,7 +4,7 @@
   "use_success": "generic_success",
   "use_failure": "generic_failure",
   "effects": [
-    "heal"
+    "heal 20"
   ],
   "power": 20,

```

Comment:
what use now is the `power` attribute?  if its part of the effects, i think we should remove it to prevent headaches down the road because it looks like it does something.

Reply:
I have no use for it, but I didn't want to just remove it without discussion. I'll get rid of it now.